lib/ansible/plugins/filter/mandatory.yml

```diff
@@ -10,11 +10,18 @@ DOCUMENTATION:
       description: Mandatory expression.
       type: raw
       required: true
+    msg:
+      description: The customized message that is printed when the given variable is not defined.
+      type: str
+      required: false
```

Comment:
was this always a parameter or should we add 'version_added'?

Reply:
Yes, this was a parameter from 2.9.